src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingService.java

```diff
@@ -57,6 +57,11 @@
      */
     public static final String ACTION_RESET = "reset";
 
+    /**
+     * Intent action to shut down the SyncthingService
+     */
+    public static final String ACTION_SHUTDOWN = "shutdown";
```

Comment:
Same point about the values of actions as the one above. Naming seems fine for these ones though.